src/index.ts

```diff
@@ -1,70 +1,109 @@
-import Renderer from './modules/Renderer';
 import Trigger from './modules/Trigger';
-import Shape from './modules/Shape';
-import {Viewport, IntializeOptions} from './@types/global';
+import Renderer from './modules/Renderer';
+import {Context, Action} from './@types/global';
+import Painter from '@/modules/Painter';
```

Comment:
It's not a big deal, but this line only use alias, it will be better to have consistency

Reply:
thanks for opinion.
i will modify it.